service/service.go

```diff
@@ -56,10 +56,17 @@ var getObservationStore = func(ctx context.Context) (observationStore *graph.DB,
 	return observationStore, errorConsumer, nil
 }
 
+var pConfig *kafka.ProducerConfig
+
+var cfg *config.Config
```

Comment:
This config is not initialised, so when it's used by getProducer it will be using default values only. Can you remove this variable and provide config as parameter accepted by `getProducuer` please? 
Then when you call getProducer from Init(), you can provide the initialised config (`svc.cfg`)